src/js/themes/base.d.ts

```diff
@@ -1039,8 +1039,13 @@ export interface ThemeType {
   };
   notification?: {
     container?: BoxProps;
+    direction?: 'column' | 'row';
+    separator?: string;
```

Comment:
Wondering if we can get away with not adding `separator` yet.

Reply:
That's fine. I'll revert to the previous approach and add a comment since it seemed that was causing confusion on the first review.